### PR TITLE
Fix: Correct LP answer column type.

### PR DESCRIPTION
**Pull request by bjones1**
This now agrees with the type from web2py / what's in the database.

### CODE REVIEW

**Review by bnmnetp — APPROVED** [review]

File: bookserver/models.py [hunk 1]
```diff
@@ -290,9 +290,9 @@ class UnittestAnswers(Base, CorrectAnswerMixin):
 @register_answer_table
 class LpAnswers(Base, AnswerMixin):
     __tablename__ = "lp_answers"
-    # See answer_. A JSON string; see RunestoneComponents for details. TODO: The length seems too short to me. Migrate this to use a ``Text`` field type instead.
-    answer = Column(String(512), nullable=False)
-    # A grade between 0 and 100. None means it the student hasn't submitted an answer yet. This was added before the ``percent`` field most other question types now have; it servers the same role, but stores the answer as a percentage. (The ``percent`` field in other questions stores values between 0 and 1.)
+    # See answer_. A JSON string; see RunestoneComponents for details.
```
Comment (bnmnetp) [inline bookserver/models.py:293]: Hmmm, should PR's such as this on have an accompanying alembic migration?

**Review by bnmnetp — APPROVED** [review]

File: bookserver/models.py (hunk 1 above)
Comment (bjones1) [reply bookserver/models.py:293]: Oddly, my database had the correct `text` type already, even though I'd used Alembic to migrate from web2py to bookserver's models. Perhaps alembic can't distinguish between a `varchar` and `text`? I ran Alembic, and it detected no changes as well. What type does do you see for answers column in `psql -c "\d lp_answers" $DBURL`?

Comment (bnmnetp) [reply bookserver/models.py:293]: My database shows `text` as the type for answer.

Comment (bnmnetp) [reply bookserver/models.py:293]: Oddly, useinfo act is declared as `varchar(512)`.   (NOT ARCHER autocorrect!!)

Comment (bjones1) [reply bookserver/models.py:293]: :) It would make more sense to have `act` be `text`...

**Pull request merged**